src/just_tls.erl

```diff
@@ -0,0 +1,166 @@
+-module(just_tls).
```

Comment:
I am starting to get lost in so many tls modules :)
There should be a doc what just_tls, ejabberd_tls and fast_tls, ssl are.

It seams, that we started to support different SSL backends :)

Reply:
there definitely will be a doc describing it. but in the next PR

Reply:
"Read the docs" style of docs can be in a separate PR.
But short description of each module in a file header should be here.